cli/create.go

```diff
@@ -61,6 +61,13 @@ func (cc *CreateCommand) addFlags() {
 	flagSet.StringVar(&cc.ipcMode, "ipc", "", "IPC namespace to use")
 	flagSet.StringVar(&cc.pidMode, "pid", "", "PID namespace to use")
 	flagSet.StringVar(&cc.utsMode, "uts", "", "UTS namespace to use")
+	flagSet.Uint16Var(&cc.blkioWeight, "blkio-weight", 0, "Block IO (relative weight), between 10 and 1000, or 0 to disable")
+	flagSet.Var(&cc.blkioWeightDevice, "blkio-weight-device", "Block IO weight (relative device weight)")
```

Comment:
Could you add an example in command line flag? So users could get how to input the blkioWeightDevice value very easily.

Reply:
`No default value' , it is no need at all

Reply:
OK, thanks a lot. I think we need to start considering encapsulate common flag handling for `pouch run` and `pouch create` into a function like `addCommonFlags(...)`. You see we have lots of duplicated flags and codes. @Ace-Tang 
Maybe in a follow-up pull request.

Reply:
you are right, common is need